controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -810,12 +849,27 @@ bool wfa_ca::get_send_1905_1_tlv_hex_list(std::list<tlv_hex_t> &tlv_hex_list,
 
                 // Validate hex notation on list of values separated by space
                 auto values = string_utils::str_split(it->second, ' ');
-                for (const auto &value : values) {
+                for (auto &value : values) {
                     if (!validate_hex_notation(value)) {
-                        err_string =
-                            "param name '" + lookup_str + "' has invalid hex notation value";
-                        return false;
+                        if (validate_mac_address(value)) {
+                            convert_mac_address_to_hex_notation(value);
+                            join = true;
+                        } else {
```

Comment:
What you did is not efficient. This solution replaces all cmdu string content just because a single value is not in hex.
Please keep here only the validation with logic AND between them.
As well, more validation are needed:
```cpp
if (!(validate_hex_notation() || validate_mac_notation() || validate_dec_notation() || validate_bin_notation())) {
    //error
}
```
On `add_tlvs_from_list()` where we iterate each value again to put it on the buffer, validate for each of the notations, and do the right conversion.